lib/filesystem/FileStream.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <iosfwd>
```

Comment:
This should be moved to global.h, (but we already implicitly include complete std::ios - to be factored out)